testing/common_testing_util.py

```diff
@@ -107,3 +121,96 @@ def printv(string, verbose=1, type='normal'):
                 print(string)
         except Exception:
             print(string)
+
+
+def assert_empty_bids_dataframe(loader_parameters: dict):
+    """Assertion function used during unit test to check if the loader parameters result in an empty bids dataframes.
+    Used as part of pytest-case scenarios for when the Step1/2/3 filters excluded almost all data. (e.g. when no
+    participants have the required Session data)
+
+    Args:
+        loader_parameters (dict): a dictionary object containing all loader parameters necessary derived from
+        config.json to construct the BidsDataFrame
+    """
```

Comment:
Function used to check empty data frame in unit test. The function comment is up to date.